entrypoint.sh

```diff
@@ -61,13 +61,13 @@ fi
 # https://groups.google.com/forum/#!topic/airbnb_airflow/4ZGWUzKkBbw
 if check_set "${ENABLE_AIRFLOW_INITDB}"; then
   echo "Initializing database for Airflow..."
-  gosu "${AIRFLOW_USER}" airflow initdb
+  gosu "${AIRFLOW_USER}" airflow initdb || gosu "${AIRFLOW_USER}" airflow db init
   echo "Database is initialized with Airflow metadata!"
 fi
 
 if check_set "${ENABLE_AIRFLOW_UPGRADEDB}"; then
   echo "Upgrading database schema for Airflow..."
-  gosu "${AIRFLOW_USER}" airflow upgradedb
+  gosu "${AIRFLOW_USER}" airflow upgradedb|| gosu "${AIRFLOW_USER}" airflow db upgrade
```

Comment:
There is a weird unicode char beside `||` lol

Reply:
Fixed, thanks for the catch!